public/client.js

```diff
@@ -272,60 +197,63 @@ socket.on('room created other', (data) => {
-	}
-	$(".error").hide();
-	var room_id = convertIntoId(data);
-	$(`#${room_id}`).remove();
-	$(`#${room_id}-msg`).remove();
+
+// destroys room because there are no users in it
+socket.on('destroy room', function(data) {
+
+    //redirect user to lobby if the active room is to be destroyed
+    if ($(".active").attr("id") == data) {
+        $("#lobby").addClass('active');
+        $("#lobby-msg").css("display", "inherit");
+    }
+
+    $(".error").hide();
+
+    var room_id = convertIntoId(data);
+    $(`#${room_id}`).remove();
+    $(`#${room_id}-msg`).remove();
 });
-//notifies when user leaves the room
-socket.on('user left room', (data) => {
-	let room_id = convertIntoId(data.room);
-	notify(`${data.username} just left room  ${data.room}`, "error");
-	$(`#${room_id}-msg`).find('.top').find('span')[1].innerHTML = data.online + ` user(s) online`;
-	$(`#${room_id}-msg`).find('.Participants').find('span')[0].innerHTML = convertIntoList(data.online_users);
+
+// notifies when user leaves the room
+socket.on('user left room', (data) =>  {
+    let room_id = convertIntoId(data.room);
+    notify(`${data.username} just left room  ${data.room}`, "error");
+    $(`#${room_id}-msg`).find('.top').find('span')[1].innerHTML = data.online + ` user(s) online`;
+    $(`#${room_id}-msg`).find('.Participants').find('span')[0].innerHTML = convertIntoList(data.online_users);
 });
-//updates info about number of users
-socket.on('update info', (rooms) => {
-	let room_id;
-	// alert(rooms);
-	for (let i = 0; i < rooms.length; i++) {
-		room_id = convertIntoId(rooms[i].name);
-		$(`#${room_id}-msg`).find('.top').find('span')[1].innerHTML = rooms[i].num_users + " user(s) online";
-		$(`#${room_id}-msg`).find('.Participants').find('span')[0].innerHTML = convertIntoList(rooms[i].users);
-	}
+
+// updates info about number of users
+socket.on('update info', (rooms)=> {
+    let room_id;
+    // alert(rooms);
+    for (let i = 0; i < rooms.length; i++) {
+        room_id = convertIntoId(rooms[i].name);
+        $(`#${room_id}-msg`).find('.top').find('span')[1].innerHTML = rooms[i].num_users + " user(s) online";
+        $(`#${room_id}-msg`).find('.Participants').find('span')[0].innerHTML = convertIntoList(rooms[i].users);
+    }
 });
-//updates info about number of users
-socket.on('room joined', function (data) {
-	const {
-		name,
-		online,
-		online_users
-	} = data;
-	var room_id = convertIntoId(name);
-	$(`#${room_id}-msg`).find('.top').find('span')[1].innerHTML = online + " user(s) online";
-	$(`#${room_id}-msg`).find('.Participants').find('span')[0].innerHTML = convertIntoList(online_users);
+
+// updates info about number of users
+socket.on('room joined', function(data) {
+    const { name, online, online_users } = data;
+    var room_id = convertIntoId(name);
+    $(`#${room_id}-msg`).find('.top').find('span')[1].innerHTML = online + " user(s) online";
+    $(`#${room_id}-msg`).find('.Participants').find('span')[0].innerHTML = convertIntoList(online_users);
```

Comment:
Don't you think that this is unnecessary(according to `.editorconfig`.

Reply:
Im unsure what you're talking about. I didn't touch that portion of code.